_source/api/logzio-public-api.yml

```
@@ -4813,3 +4932,369 @@ definitions:
+          
+          
+          The fields must be ordered pairs of the group by fields already in use in the `queryDefinition`.
+        items:
+          type: object
+          additionalProperties:
+          example: {0: "region", 1: "region"}
+  SubAlertOutput:
+    type: object
+    description: Selects the data output to be sent in the notification when the alert triggers. Not applicable, when grouping by fields or aggregating results, as the output is auto-selected.
+    properties:
+      columns:
+        type: array
+        items:
+          $ref: '#/definitions/ColumnConfig'
+      shouldUseAllFields:
+        type: boolean
+        description: If `true`, the notification ouput will include entire logs with all of their fields in the sample data.
+        default: true
+  AlertV2Request:
+    type: object
+    required:
+    properties:
+      title:
+        type: string
+        description: Alert title
+        example: Excessive WARN levels in PROD
+      description:
+        type: string
+        description: A description of the event, its significance, and suggested next steps or instructions for the team.
+        example: Steps to remediate...
+      tags:
+        type: array
+        items:
+          type: string
+        maxItems: 10
+        minItems: 0
+        description: Tags for filtering alerts and triggered alerts. Can be used in Kibana Discover, dashboards, and more.
+        example: network
+      output:
+        $ref: '#/definitions/AlertOutput'
+      searchTimeFrameMinutes:
+        type: integer
+        minimum: 5
+        maximum: 1440
+        format: int32
+        description: >-
+          The time frame for evaluating the log data is a sliding window, with 1 minute granularity.
+          
+          
+          The recommended minimum and maximum values are not validated, but needed to guarantee the alert's accuracy.
+
+
+          The minimum recommended time frame is 5 minutes, as anything shorter will be less reliable and unnecessarily resource-heavy.
+
+
+          The maximum recommended time frame is 1440 minutes (24 hours). The alert runs on the index from today and yesterday (in UTC) and the maximum time frame increases throughout the day, reaching 48 hours exactly before midnight UTC.
+        example: 20
+      subComponents:
+        type: array
+        description: Sets the search criteria using a search query, filters, group by aggregations, accounts to search, and trigger conditions.
+        items:
+          $ref: '#/definitions/SubAlert'
+      correlations:
+        $ref: '#/definitions/SubAlertCorrelation'
+      enabled:
+        type: boolean
+        description: If `true`, the alert is enabled and active.
+  AlertTrigger:
+    type: object
+    description: Sets an additional triggering threshold and severity tab to label the event when the alert triggers.
```

it is **the** condition for which the alert triggers, so the word "additional" makes it sound not required somehow.
```suggestion
    description: Sets the triggering threshold and severity tab to label the event when the alert triggers.
```

---

fixed